diff --git a/node_modules/@react-native-ml-kit/face-detection/android/src/main/java/com/rnmlkit/facedetection/FaceDetectionModule.java b/node_modules/@react-native-ml-kit/face-detection/android/src/main/java/com/rnmlkit/facedetection/FaceDetectionModule.java
index 3c6f7cf..c7d16eb 100644
--- a/node_modules/@react-native-ml-kit/face-detection/android/src/main/java/com/rnmlkit/facedetection/FaceDetectionModule.java
+++ b/node_modules/@react-native-ml-kit/face-detection/android/src/main/java/com/rnmlkit/facedetection/FaceDetectionModule.java
@@ -313,13 +313,11 @@ public class FaceDetectionModule extends ReactContextBaseJavaModule {
                             new OnFailureListener() {
                                 @Override
                                 public void onFailure(@NonNull Exception e) {
-                                    e.printStackTrace();
-                                    promise.reject("Face detection failed", e);
+                                    promise.reject("E_FACE_DETECTION_FAILED", "Face detection failed.");
                                 }
                             });
         } catch (IOException e) {
-            e.printStackTrace();
-            promise.reject("Face detection failed", e);
+            promise.reject("E_FACE_IMAGE_UNREADABLE", "Face detection failed.");
         }
     }
 }
